SPECS/vitess/vitess.spec

```diff
@@ -3,7 +3,7 @@
 
 Name:           vitess
 Version:        16.0.2
-Release:        6%{?dist}
+Release:        7%{?dist}
```

Comment:
%check section is failing, `go check` isn't a real command. Likely unrelated to these changes.